cmd/fetch_policy.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Define the `ec fetch policy` command
+package cmd
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/hacbs-contract/ec-cli/internal/policy/source"
+	"github.com/hacbs-contract/ec-cli/internal/utils"
+)
+
+func fetchPolicyCmd() *cobra.Command {
+	var (
+		sourceUrls []string
+		destDir    string
+		useWorkDir bool
+	)
+
+	cmd := &cobra.Command{
+		Use:   "policy --source <source-url>",
+		Short: "Fetch policy rules from a git repository or other source",
+		Long: `Fetch policy rules (rego files) from a git repository or other source.
+
+Each policy source will be downloaded into a separate unique directory inside
+the "policy" directory under the destination directory specified. The
+destination directory is either an automatically generated temporary work dir
+if --work-dir is set, the directory specified with the --dest flag, or the
+current directory if neither flag is specified.
+
+This command is based on 'conftest pull' so you can refer to the conftest pull
+documentation for more usage examples and for details on the different types of
+supported source URLs.
+
+Note that this command is not typically required to verify the Enterprise
+Contract. It has been made available for troubleshooting and debugging
+purposes.`,
+
+		Example: `Fetching policies from multiple sources to a specific directory:
+
+  ec fetch policy --dest fetched-policies \
+    --source github.com/hacbs-contract/ec-policies//policy/lib \
```

Comment:
Out of curiosity, how do you specify a branch?

Reply:
You can add params, e.g. `?ref=main`